src/perl/lib/wtsi_clarity/epp/reports/sample_qc_report.pm

```diff
@@ -92,7 +92,7 @@ sub headers {
     'Sample UUID',
     'Concentration',
     'Sample volume',
-    'Library concentration',
+    'Library molarity',
     'DNA amount library prep',
     'Status',
   ]
```

Comment:
We could DRY this method to return the keys of the hash the _build_row returns, then you don't have the report header twice in the code.